src/gi-stubs/repository/_Gdk4.pyi

```diff
@@ -2306,7 +2307,7 @@ _overrides_module = ...  # FIXME Constant
 _version: str = "4.0"
 
 def cairo_draw_from_gl(
-    cr: cairo.Context,
+    cr: cairo.Context[_SomeSurface],
```

Comment:
_SomeSurface needs to be definied somewhere in this module

Reply:
I modified the generator to automatically add the necessary `TypeVar` if `cairo` is used.